app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/campaign/Repository/ActiveCampaignRepository.kt

```diff
@@ -0,0 +1,61 @@
+package org.wikiedufoundation.wikiedudashboard.ui.campaign.Repository
+
+import android.widget.Toast
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import kotlinx.coroutines.*
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.dao.ActiveCampaignDao
+import org.wikiedufoundation.wikiedudashboard.data.network.WikiEduDashboardApi
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.data.CampaignListData
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.data.CampaignResponse
+import timber.log.Timber
+
+/**Declares the DAO as a private property in the constructor. Pass in the DAO
+*instead of the whole database, because you only need access to the DAO*
+ * */
+class ActiveCampaignRepository(private val wikiEduDashboardApi: WikiEduDashboardApi,
```

Comment:
Change package Name Repository -> repository. 

package-name should not contain camelCase or snake_case letters. 

https://stackoverflow.com/a/3179277